sasmodels/direct_model.py

```diff
@@ -330,7 +329,9 @@ def _calc_theory(self, pars, cutoff=0.0):
 
         # Need to pull background out of resolution for multiple scattering
         default_background = self._model.info.parameters.common_parameters[1].default
```

Comment:
This doesn't seem to work for me... I still get default background from model but I probably don't know how to properly fit sesans data. Is it enough to load *ses file and send it to fitting?

Reply:
The background for the SAS calculations is standard added for all models. This is inconvenient for the SESANS, where the background standard should be 0. I don't know how this will affect the sascomp simulator. It would be convenient to be able to use the sascomp simulator, but indeed we can manage at the moment pretty well by just first reading in any *ses file.

Reply:
DirectModel is not on the code path for sasview, so it won't affect you when running from the GUI. If you are using the command line sascomp tool or running the sesans fits from sasmodels/examples then you should not have any background added. If I print the background value then I get 0.0 for sesans models, as expected.